test/controller/beast.test.ts

```diff
@@ -0,0 +1,113 @@
+import { Request, Response } from 'express';
+import BeastController from '../../src/controller/beast';
+import BeastModel from '../../src/models/beast';
+import { BadRequestError, UnknownError } from '../../src/models/errors';
+
+jest.mock('../../src/models/beast');
+
+const mockResponse = () => {
+    const res = {} as Response;
+    res.status = jest.fn().mockReturnValue(res);
+    res.json = jest.fn().mockReturnValue(res);
+    return res;
+};
+
+const mockRequest = (body: Record<string, any> = {}, params: Record<string, any> = {}) => ({
```

Comment:
Can we move the mockRequest & response into a test helper file? I feel like we can reuse these across multiple test files